src/components/DropDown/DropDown.tsx

```diff
@@ -40,6 +40,17 @@ const DropDown: FC<Props> = ({
 
   const assigned = pathwayIsAssigned(patientRecords, evaluatedPathway?.pathway);
 
+  const formatOptionLabel = (option: Readonly<Option>): object => (
+    <div>
+      <div className={styles.name}> {option.label} </div>
+      <div className={styles.check}>
```

Comment:
This div is also unnecessary. I would remove it and add the class directly to the `FontAwesomeIcon` component. Then you can delete the styles you have for `check` and just add `margin-left: 5px;`. You can also get rid of the `<div>` tag for the label, which would make the `name` class name with `display: inline-block;` unnecessary since the label is treated as an inline-block by default.

```
const formatOptionLabel = (option: Readonly<Option>): object => (
  <>
    {option.label}
    {pathwayIsAssigned(patientRecords, option.value.pathway) && (
      <FontAwesomeIcon icon={faCheckCircle} className={styles.check} />
    )}
  </>
);
```

```
.check {
  margin-left: 5px;
}
```